lib/next-amp-demo/pages/components/AmpBind.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {AmpImg, AmpCarousel} from '@ampproject/toolbox-next-amp/src-gen';
+export const config = {amp: true};
+
+export default () => (
+  <>
+    <h1>amp-bind</h1>
+    <AmpCarousel bindSlide='slide' width='400' height='300' type='slides'>
+      <AmpImg src='https://picsum.photos/id/231/400/300' layout='fill'></AmpImg>
+      <AmpImg src='https://picsum.photos/id/232/400/300' layout='fill'></AmpImg>
+      <AmpImg src='https://picsum.photos/id/233/400/300' layout='fill'></AmpImg>
+    </AmpCarousel>
+    <button
+      on='tap:AMP.setState({
```

Comment:
Do we care about indentation of multi-line attributes? Prettier won't touch them unfortunately.

Reply:
I personally care a lot about these. Would be great if we could find some kind of standard here.